tests/integration/actions/collections/base.py

```diff
@@ -58,6 +58,8 @@ def test(
         comment,
     ):
         # pylint: disable=too-many-arguments
+        # pylint: disable=too-many-locals
+
```

Comment:
Based on the style used in other places, this extra empty line is accidental + it violates PEP 257 so why not drop it
```suggestion
```